data/jobs.txt

```diff
@@ -1477,6 +1477,74 @@ mission "Core Pirate Attack"
 	on complete
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
+		
+mission "Raider Attack 1"
+	name "Defend <planet>"
+	description "Defeat an alien raid on <destination>."
+	minor
+	repeat
+	to offer
+		"combat rating" > 550
+		random < 7
+	source
+		planet Sheratan
+		planet Shangri-la
```

Comment:
Quote non-reserved keywords, e.g. "Shangri-La"
Also, the `l` needs to be `L` as `"Shangri-la"` is not the planet, `"Shangri-La"` is.